pkg/storaged/anaconda.jsx

```diff
@@ -96,8 +153,22 @@
     const mpm = { };
     for (const p in client.blocks) {
         const b = client.blocks[p];
-        mpm[decode_filename(b.Device)] = block_info(b);
+        mpm[device_name(b)] = block_info(b);
+    }
+
+    // Add inactive logical volumes
+    for (const vg_p in client.vgroups) {
+        const vg = client.vgroups[vg_p];
+        for (const lv of client.vgroups_lvols[vg_p] || []) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19924-20240207-125230-6cf61ca9-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)